pkg/leakybucket/manager_load.go

```diff
@@ -160,6 +161,9 @@ func LoadBuckets(files []string, dataFolder string) ([]BucketFactory, chan types
 			g.Filename = filepath.Clean(f)
 			g.BucketName = seed.Generate()
 			g.ret = response
+			if g.Scope == "" {
+				g.Scope = "IP"
```

Comment:
do we want to maintain a list of "native" scopes (ie. ip/range/..) in a iota/enum of some sorts ? (avoid typos and make scope-changes easier)